Library/Core/Uno.Permissions/Permissions.java

```diff
@@ -76,7 +76,11 @@ private static void nextRequest()
             _currentRequest.requestID);
     }
 
-    public static void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
+    public static void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) 
+    {
+        if( _currentRequest != null || _requests.size() == 0) 
```

Comment:
Hmm, I'm not sure if we need that `_requests.size() == 0`-check... The code below doesn't seem to depend on this variable at all...

Reply:
Humm, I see .

It changed now, thanks for your review